win32.mak

```diff
@@ -6,16 +6,17 @@ defaulttarget:
 	cd ..
 
 auto-tester-build:
```

Comment:
I assume these changes will break the makefile patching done on the auto-tester?

If that is the case:
- Remove these changes
- Forward [`MAKECMDGOALS`](https://www.gnu.org/software/make/manual/html_node/Goals.html) to `test/Makefile` and do an early exit there

Reply:
Yeah I thought the windows 64 diff had been disabled, but I was wrong :/

https://github.com/braddr/at-client/blob/master/src/do_fixup.sh#L21